generators/php/php.js

```diff
@@ -59,13 +60,28 @@ export const _toPhp = request => {
       }
       requestDataCode += ']'
     } else if (request.isDataBinary) {
-      requestDataCode += "file_get_contents('" + quote(request.data.substring(1)) + "')"
+      requestDataCode = "file_get_contents('" + quote(request.data.substring(1)) + "')"
```

Comment:
I think this needs to also be wrapped in a `if (request.data.charAt(0) === '@') {`, otherwise pass the data as-is.